acceptance/acceptance_test.go

```diff
@@ -678,44 +678,59 @@ func testAcceptance(
 						t.Log("app is runnable")
 						assertMockAppRunsWithOutput(t, repoName, "Launch Dep Contents", "Cached Dep Contents")
 
+						version, err := packVersion(packPath)
+						h.AssertNil(t, err)
+						packSemver = semver.MustParse(strings.TrimPrefix(strings.Split(version, " ")[0], "v"))
+						packSupport := packSemver.GreaterThan(semver.MustParse("0.10.0")) || packSemver.Equal(semver.MustParse("0.0.0"))
+
 						t.Log("restores the cache")
-						if lifecycleDescriptor.Info.Version.LessThan(semver.MustParse("0.6.0")) {
-							h.AssertContainsMatch(t, output, `(?i)\[restorer] restoring cached layer 'simple/layers:cached-launch-layer'`)
-							h.AssertContainsMatch(t, output, `(?i)\[analyzer] using cached launch layer 'simple/layers:cached-launch-layer'`)
-						} else {
+						if semver.MustParse(lifecycleDescriptor.API.PlatformVersion.String()).LessThan(semver.MustParse("0.3")) || !packSupport {
 							h.AssertContainsMatch(t, output, `(?i)\[restorer] Restoring data for "simple/layers:cached-launch-layer" from cache`)
 							h.AssertContainsMatch(t, output, `(?i)\[analyzer] Restoring metadata for "simple/layers:cached-launch-layer" from app image`)
+						} else {
+							h.AssertContainsMatch(t, output, `(?i)\[creator] Restoring data for "simple/layers:cached-launch-layer" from cache`)
+							h.AssertContainsMatch(t, output, `(?i)\[creator] Restoring metadata for "simple/layers:cached-launch-layer" from app image`)
 						}
 
 						t.Log("exporter reuses unchanged layers")
-						h.AssertContainsMatch(t, output, `(?i)\[exporter] reusing layer 'simple/layers:cached-launch-layer'`)
-
-						if lifecycleDescriptor.Info.Version.LessThan(semver.MustParse("0.6.0")) {
-							t.Log("cacher reuses unchanged layers")
-							h.AssertContainsMatch(t, output, `(?i)\[cacher] reusing layer 'simple/layers:cached-launch-layer'`)
+						if semver.MustParse(lifecycleDescriptor.API.PlatformVersion.String()).LessThan(semver.MustParse("0.3")) || !packSupport {
+							h.AssertContainsMatch(t, output, `(?i)\[exporter] reusing layer 'simple/layers:cached-launch-layer'`)
 						} else {
-							h.AssertContainsMatch(t, output, `(?i)\[exporter] Reusing cache layer 'simple/layers:cached-launch-layer'`)
+							h.AssertContainsMatch(t, output, `(?i)\[creator] reusing layer 'simple/layers:cached-launch-layer'`)
 						}
 
-						t.Log("rebuild with --clear-cache")
-						output = h.Run(t, subjectPack("build", repoName, "-p", appPath, "--clear-cache"))
-						h.AssertContains(t, output, fmt.Sprintf("Successfully built image '%s'", repoName))
+						// TODO: check if we need the following test - no cacher exists
```

Comment:
I'm not too familiar with this code but it looks like we might need three cases here? One for platform api version 0.2 that uses `cacher`, one that uses `exporter`, and one that uses `creator`? The `cacher` one might be too old at this point, maybe?

Reply:
When running `make acceptance-all` on the main branch, we don't get to the `cacher` in any configuration so I deleted this part because I guess it's too old (kept only the `exporter` and the `creator`.